Kickstarter-iOS/Views/Controllers/FacebookConfirmationViewController.swift

```diff
@@ -9,6 +9,7 @@ import UIKit
 
 internal final class FacebookConfirmationViewController: UIViewController,
   MFMailComposeViewControllerDelegate {
+  @IBOutlet var disclaimerButton: UIButton!
```

Comment:
Did the ticket include any requirements for a new button? I thought the ticket was just an update to the copy?

Reply:
Yes it did, a disclaimer copy. I realised a button was used on the Signup Screen so that it shows an action sheet when tapped, I thought the same behaviour applies here.

Reply:
I'm interpreting the task differently and would ping Vicki to verify. This appears to be only copy updates and I don't see anywhere indicating the addition of a new button.

Reply:
It's an addition not an update:
> The interstitial screen after returning from Facebook authorization has this legal copy below the Sign up button: By signing up, you agree to our Terms of Use, Privacy Policy, and Cookie Policy. Our policies explain how we use your data to deliver, improve, and promote our service and our site, and how you can exercise your rights to control that use.

You should look at the screenshots attached to the NT-418 epic. There was previously no disclaimer below the "Create new account" button

Reply:
I think you are misinterpreting the task. The PR should only be addressing the requirements listed in `NT-419`. The epic it belongs to is quite old and needs updating. But again, please verify with both Vicki and Jonathan (for design related questions).